telemetry/definitions/commonDefinitions.json

```diff
@@ -2174,6 +2180,154 @@
                 }
             ]
         },
+        {
+            "name": "app_openLocalInvokeWebView",
+            "description": "Click open Local invoke WebView button in Application Builder",
+            "metadata": [
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "source",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "app_openRemoteInvokeWebView",
```

Comment:
It's very good to avoid verbosity, but in this case, `app_` may be a bit too generic. When stakeholders are building dashboards and exploring metrics, the prefix `app_` looks like it could refere to the IDE or Toolkit project itself, not a specific vertical. 

Would `samApp_` or `cloudApp_` or some other name be possible?

Reply:
Yeah we could use `appBuilder_` I believe. Could you help to confirm @seshubaws

Reply:
Yes let's use `appBuilder_`